### ISSUE TITLE
Cannot Paste From Keyboard On Mac When Command Key Remapped to Control Key

### ISSUE THREAD

**Issue by hegemony2** [a85/POSTMan-Chrome-Extension#3739]
## App Details:
Postman for Mac
Version 5.3.2 (5.3.2)
OS X 10.11.6 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: For about 4 months
 2. Expected behaviour:  I should be able to paste into the postman app
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app):  No logs occur when attempting to paste
 4. Screenshots (if applicable):  see attached on modification to keyboard
![keyboard_and_postman](https://user-images.githubusercontent.com/15934084/32462666-08e3e3c6-c2ef-11e7-97b5-04c3d24b9f5d.png)


## Steps To Reproduce:
1. (MAC) Go to System Preferences... / Keyboard / Modifier Keys...
2. Map the Control Key to Command and Map the Command Key to Control
3. copy anything to clipboard on mac
4. attempt to paste into, say, body within postman using keyboard (ctrl-v).
5. nothing happens, even though edit menu blinks.

**Comment by sdnts** [comment]
Hi @hegemony2, I did what you said, but I'm unable to see this.
I was able to copy and paste. Can you check if you can paste in any other application?

**Comment by hegemony2** [comment]
Yes, it works in other applications.  It also works when I map the keys back to their original function.

I was able to get it to work…  I had an App Shortcut for the keyboard that maps ctrl-v to “Paste and Match Style”…  when I deleted and re-added that to my system/keyboard preferences, I can now paste in postman:

[cid:image001.png@01D357B9.48467660]

Perhaps upon installation, postman is not picking up this preference?  Seems odd, but it works now.  Thanks for your help.
